pkg/questionModel/questionModel.go

```diff
@@ -0,0 +1,208 @@
+package questionModel
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"simple-ec2/pkg/ec2helper"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/list"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/glamour"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/olekukonko/tablewriter"
+)
+
+const defaultWidth = 20
+
+var (
+	noStyle           = lipgloss.NewStyle()
+	xSmallLeftPadding = lipgloss.NewStyle().PaddingLeft(1)
+	smallLeftPadding  = lipgloss.NewStyle().PaddingLeft(3)
+	mediumLeftPadding = lipgloss.NewStyle().PaddingLeft(5)
+	largeLeftPadding  = lipgloss.NewStyle().PaddingLeft(7)
+	xLargeLeftPadding = lipgloss.NewStyle().PaddingLeft(9)
+	focused           = lipgloss.NewStyle().Foreground(lipgloss.Color("170"))
```

Comment:
Can you add an inline comment to each `lipgloss.Color()` statement with the color name?